cookbook/bundles/best_practices.rst

```diff
@@ -72,49 +68,44 @@ class name.
     :class:`Symfony\\Bundle\\FrameworkBundle\\FrameworkBundle`.
 
 Each bundle has an alias, which is the lower-cased short version of the bundle
-name using underscores (``acme_hello`` for ``AcmeHelloBundle``, or
-``acme_social_blog`` for ``Acme\Social\BlogBundle`` for instance). This alias
-is used to enforce uniqueness within a bundle (see below for some usage
-examples).
+name using underscores (``acme_blog`` for ``AcmeBlogBundle``). This alias
+is used to enforce uniqueness within a bundle (see below for some usage examples).
 
 Directory Structure
 -------------------
 
-The basic directory structure of a HelloBundle must read as follows:
+The basic directory structure of a AcmeBlogBundle must read as follows:
 
 .. code-block:: text
 
-    XXX/...
-        HelloBundle/
-            HelloBundle.php
-            Controller/
-            Resources/
-                meta/
-                    LICENSE
-                config/
-                doc/
-                    index.rst
-                translations/
-                views/
-                public/
-            Tests/
-
-The ``XXX`` directory(ies) reflects the namespace structure of the bundle.
-
-The following files are mandatory:
-
-* ``HelloBundle.php``;
+    <your-project>/
+    ├─ ...
+    └─ src/
+       └─ Acme/
+          └─ BlogBundle/
```

Comment:
why showing an application? I'm -1 for developing a bundle inside an application, we should only show the bundle structure (so `AcmeBlogBundle.php` is in the root of the structure)